skxray/core/correlation.py

```diff
@@ -405,3 +404,338 @@ def auto_corr_scat_factor(lags, beta, relaxation_rate, baseline=1):
     return beta * np.exp(-2 * relaxation_rate * lags) + baseline
 
 
+two_time_results = namedtuple(
+    'two_timecorrelation_results',
+    ['two_time', 'two_time_internal_state']
+)
+
+_two_time_internal_state = namedtuple(
```

Comment:
Is there any reason why the internal state object also cannot be shared between one-time and two-time?